src/test/java/org/mastodon/mamut/treesimilarity/tree/BranchSpotTreeExamples.java

```diff
@@ -66,10 +64,8 @@ public static BranchSpotTree emptyTree()
 	 *                                         branchSpot3(lifespan=30)
 	 * </pre>
 	 */
-	public static BranchSpotTree tree1()
+	public static BranchSpotTree tree1( final Model model )
```

Comment:
Okay. I dropped the commit where I made these changes. I also do not need these changes anymore, since increasing the Ram for the tests to 1gb resolved the problems already.